src/logion/model/locrequest.model.ts

```diff
@@ -829,6 +838,12 @@ export class LocRequestAggregateRoot {
         return polkadotAccount(this.ownerAddress || "");
     }
 
+    canOpen(user: SupportedAccountId | undefined): boolean {
```

Comment:
Given the way this method is used right now, the implementation is OK but a test on the request's status should be added in order to be future-proof.